CHANGELOG.md

```diff
@@ -2,15 +2,31 @@
 
 ### Bugfixes
 
-* Lorem ipsum.
+* Fixes a bug in chuncked binary column returning null value. 
+  Issue [#2418](https://github.com/realm/realm-core/pull/2418).
```

Comment:
The _issue_ link should be `https://github.com/realm/realm-core/issues/2418`, but I would still suggest that we primarily note the PR number as it is the canonical "pointer" for the actual changes to the source.